### PR TITLE
Updated docs to reflect time series additions

### PR DESCRIPTION
**Pull request by ojustino**
I documented the additions I made in #187 -- time series capabilities in `TableLayer`s and `wwt.get_current_time()`.

### CODE REVIEW

File: docs/layers.rst [hunk 1]
```diff
@@ -28,7 +28,8 @@ first example. We start off by using `astropy.table
 comma-separated values (CSV) file of the data::
 
     >>> from astropy.table import Table
-    >>> OEC = 'https://worldwidetelescope.github.io/pywwt/data/open_exoplanet_catalogue.csv'
+    >>> OEC = 
+'https://worldwidetelescope.github.io/pywwt/data/open_exoplanet_catalogue.csv'
```
Comment (pkgw) [inline docs/layers.rst]: I'm pretty sure this linewrap will break the doc formatting here; it should be undone.

File: docs/layers.rst [hunk 2]
```diff
@@ -55,13 +56,15 @@ Be sure to zoom in so that you can see the Earth properly. Next, we use a
 dataset that includes all recorded earthquakes in 2010::
 
     >>> from astropy.table import Table
-    >>> EARTHQUAKES = 'https://worldwidetelescope.github.io/pywwt/data/earthquakes_2010.csv'
+    >>> EARTHQUAKES = 
+'https://worldwidetelescope.github.io/pywwt/data/earthquakes_2010.csv'
```
Comment (pkgw) [inline docs/layers.rst]: Ditto.

File: docs/layers.rst [hunk 3]
```diff
@@ -55,13 +56,15 @@ Be sure to zoom in so that you can see the Earth properly. Next, we use a
...
     >>> table = Table.read(EARTHQUAKES, delimiter=',', format='ascii.basic')
 
 We can then add the data layer using::
 
     >>> layer = wwt.layers.add_table_layer(table=table, frame='Earth',
-    ...                                    lon_att='longitude', lat_att='latitude')
+    ...                                    lon_att='longitude', 
+lat_att='latitude')
```
Comment (pkgw) [inline docs/layers.rst]: Ditto.

File: docs/layers.rst [hunk 4]
```diff
@@ -125,8 +128,45 @@ Finally, it is possible to set the units for the longitude::
...
+:class:`~astropy.time.Time` objects, :class:`~datetime.datetime` objects,  or 
+`ISOT  
+<https://docs.astropy.org/en/stable/api/astropy.time.TimeISOT.html#astropy.time.
+TimeISOT>`_ compliant strings.)    
+
+Once the time in the viewer matches an object's stated time in the table, its
+corresponding point will pop into view. See :ref:`views` for more information
+on how to control time in the viewer.
+
+By default, time series points disappear a certain amount of time after they
```
Comment (pkgw) [inline docs/layers.rst]: This should specify that the default fading time is 15 days (in WWT-time, not real-world time!)

File: docs/settings.rst [hunk 5]
```diff
@@ -28,7 +28,7 @@ by the :func:`matplotlib.colors.to_hex` function from Matplotlib::
     >>> wwt.constellation_figure_color = '#D3BC8D'
     >>> wwt.constellation_selection_color = (1, 0, 1)
 
-Numerical settings all take values as Astropy :class:`astropy.units.Quantity`
+Numerical settings all take values as astropy :class:`astropy.units.Quantity`
```
Comment (pkgw) [inline docs/settings.rst]: Looking at the rest of the docs, it doesn't look like we're fully consistent about this, but Astropy should stay capitalized when it refers to the project and not the specific `astropy` Python module.

File: docs/views.rst [hunk 6]
```diff
@@ -119,7 +119,27 @@ You can then resume the passage of time with the
...
+    >>> from astropy.time import Time
+    >>> my_time = Time('2017-03-09')
+    >>> wwt.set_current_time(my_time)
+    
+(To bring the viewer back to real time, call 
+:meth:`~pywwt.BaseWWTWidget.set_current_time` without an argument.)
+
+If you're unsure of where in time you've landed, the
+:meth:`~pywwt.BaseWWTWidget.get_current_time` method provides a printout of the
+viewer's current time in UTC::
```
Comment (pkgw) [inline docs/views.rst]: The printout only happens in an interactive environment where you don't assign the value to a variable. This should read "the get_current_time method returns the viewer's current time as an :class:`astropy.time.Time` object in the UTC timezone.

File: docs/layers.rst (hunk 1 above)
Comment (ojustino) [reply docs/layers.rst]: Changed.

File: docs/layers.rst (hunk 2 above)
Comment (ojustino) [reply docs/layers.rst]: Changed.

File: docs/layers.rst (hunk 3 above)
Comment (ojustino) [reply docs/layers.rst]: Changed.

File: docs/layers.rst (hunk 4 above)
Comment (ojustino) [reply docs/layers.rst]: I'll specify, though it's 16 days in both the SDK and `layers.py`.

File: docs/settings.rst (hunk 5 above)
Comment (ojustino) [reply docs/settings.rst]: Changed.

File: docs/views.rst (hunk 6 above)
Comment (ojustino) [reply docs/views.rst]: Mostly adopted, but I thought that "in (the) UTC timezone" was redundant (e.g. "EDT time") and just saying "in UTC" seems to be common. If you feel strongly about it I can push another commit.

**Comment by ojustino** [comment]
Trying to pass some checks.

**Comment by pkgw** [comment]
@ojustino We can ignore the failures related to the `requests` URLs; it's an issue that's likely transient and on their side. If all of the other tests are passing then things are OK.

**Pull request merged**